### DISCUSSION TITLE
Reduced Regression Model - support for exogenous variables?

### DISCUSSION THREAD

**Discussion by ngupta23** [alan-turing-institute/sktime#2184] [Q&A]
Since scikit-learn regressors support features (X), I was wondering whether the reduced regression models using these regressors also support exogenous variables (beyond the lagged y values)? If so, then the tag `ignores-exogeneous-X` may not be set appropriately.

https://github.com/alan-turing-institute/sktime/blob/87bdf36dbc0990f29942eb6f7fa56a8e6c5fa7b7/sktime/forecasting/compose/_reduce.py#L343

**Comment by fkiraly** [comment]
Yes, I've looked and think this was wrong.
Fix is here:
https://github.com/alan-turing-institute/sktime/pull/2230
